{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,24 +1,17 @@
 [tox]
-envlist = py34-coverage
+envlist = py34
 skipsdist = True
 
-[base]
-
 [testenv]
-setenv =
-    DJANGO_SETTINGS_MODULE={{ cookiecutter.project_slug }}.settings.base
-    PYTHONPATH={toxinidir}
-    SECRET_KEY=test
-    DATABASE_URL=postgresql://{{ cookiecutter.project_slug }}:{{ cookiecutter.project_slug }}@localhost/{{ cookiecutter.project_slug }}
 commands=
-    django-admin.py test --noinput []
-deps = -r{toxinidir}/requirements/base.txt
+    ./run_pytest.py --cov-report=term --cov={{ cookiecutter.project_slug }} -n 2
+deps = -r{toxinidir}/requirements/dev.txt
+
+[pytest]
+DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.settings.dev
```

Comment:
I think the test env should either use a dedicated test settings file, or the base one.